src/config/machines/example-grant-machine.js

```diff
@@ -155,38 +156,57 @@ export const exampleGrantMachine = createMachine({
       }
     },
 
-    'second-question': {
+    consent: {
       on: {
         BACK: {
           target: 'country',
           actions: ['updateCurrentPageId']
         },
         NEXT: {
-          target: 'final',
+          target: 'confirmation',
           actions: ['trackPageCompletion', 'updateCurrentPageId', 'updateAnswers']
         }
       },
       meta: {
-        currentPageId: 'second-question', // NOSONAR:S1192 - need to improve this later
+        templateId: 'consent-confirmation',
+        currentPageId: 'consent', // NOSONAR:S1192 - need to improve this later
         previousPageId: 'country',
-        nextPageId: 'final',
+        nextPageId: 'confirmation',
+        messageHeader: 'Confirm and send',
+        messageContent: `I confirm that, to the best of my knowledge, the details I have provided are correct.</br></br>
```

Comment:
Inline definition of html within config was one of the markers that made Cap's code feel bloated. Do we want to be doing that here? Happy if we have plans to refactor later.

Reply:
Let's talk on a huddle.

Reply:
Applied